.github/pr-labeler-config.yml

```diff
@@ -0,0 +1,46 @@
+backend:
+# TODO
```

Comment:
@tenthe @dominikriemer what do you think we should provide here?

Reply:
Not sure if it's possible but we could filter for all modules starting with streampipes-* except streampipes-extensions and non-java modules